src/Bloc/BlSpaceManager.class.st

```diff
@@ -91,3 +91,15 @@ BlSpaceManager >> spaces [
 
 	^ spaces
 ]
+
+{ #category : #'startup - shutdown' }
+BlSpaceManager >> start [
+
+	spaces do: [ :aSpace | aSpace ensureWindowOpen ]
```

Comment:
This class has the only sends of ensureWindowOpen and rememberVisibleStatus. Maybe previousVisibleStatus flag fits better here in BlSpaceManager instead of BlSpace. It is here where this previousVisibleStatus is stored and restored.

Reply:
The visible status is of the window (aka the space) if we put the flag in the space manager we need a collection of flags (one per each space).

Reply:
Thanks for the review. We need to store the state of the open windows if we want to reopen in the same place.